### PR TITLE
Bugfix/other apps not building with svg as component

### PR DESCRIPTION
**Pull request by 0xRowdy**
## GitHub Issue

Add a link to the GitHub issue.

Screen Shot 2022-09-06 at 1 27 32 PM

## Changes

A summary of the changes in the code in this PR. If making any UI changes consider adding screenshots or gifs.

## Packages Added

Brief list of any packages added, and if folks need to rerun `yarn install` to run locally

## Checks

Before making your PR, please check the following:

- [ ] Critical lint errors are resolved
- [ ] App runs locally
- [ ] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: apps/core-app/src/pages/Dao.tsx [hunk 1]
```diff
@@ -2,12 +2,34 @@ import { useParams, Outlet } from 'react-router-dom';
...
 
 export function Dao() {
   const { daochain, daoid } = useParams();
   const { provider } = useHausConnect();
   const { dao } = useDao();
 
+  const apps = {
+    trigger: {
+      name: 'Core App',
+      url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
```
Comment (brossetti1) [inline apps/core-app/src/pages/Dao.tsx:15]: can you check that this is the right link @skuhlmann ?

File: libs/daohaus-connect-feature/src/components/NetworkButton/NetworkButton.tsx [hunk 2]
```diff
@@ -35,12 +35,12 @@ export const getNetworkPanels = (
...
   switchNetwork: (id: string) => void
 ) =>
-  Object.values(availableNetworks).map((network) => {
+  Object.values(availableNetworks).map((network, i) => {
     const handleNetworkSwitch = () => {
       switchNetwork(network.chainId);
     };
     return (
-      <DropdownMenuItem asChild>
+      <DropdownMenuItem key={i} asChild>
```
Comment (brossetti1) [inline libs/daohaus-connect-feature/src/components/NetworkButton/NetworkButton.tsx:43]: fixes a missing key error

File: apps/core-app/src/pages/Dao.tsx (hunk 1 above)
Comment (brossetti1) [reply apps/core-app/src/pages/Dao.tsx:15]: fyi - this url is not active, its the trigger for the dropdown -- it just cant be blank

Comment (skuhlmann) [reply apps/core-app/src/pages/Dao.tsx:15]: @brossetti1 lgtm

**Review by skuhlmann — APPROVED** [review]

**Pull request merged**